.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2
+jobs:
+  build_and_test:
+    docker:
+      - image: ros:foxy
+    steps:
+      - checkout
+      - run:
+          name: Set Up Container
+          command: |
+            apt update -qq && apt install -y build-essential cmake python3-colcon-common-extensions python3-rosdep libeigen3-dev
+            apt upgrade -y
+            source `find /opt/ros -maxdepth 2 -name local_setup.bash | sort | head -1`
+            mkdir -p src/grid_map && mv `find -maxdepth 1 -not -name . -not -name src` src/grid_map/
+            rosdep update
+            rosdep install -y --ignore-src --from-paths src
```

Comment:
Should we comment it out for now, till we start needing it, only specifying the packages needing it as we go, and at the end of the port return it to normal?

Reply:
^ Thats only for line 16

Reply:
```suggestion
```

Reply:
No, we still need to install dependencies it requires since the docker container will be completely empty other than a basic ROS install. It needs to install deps to build